extensions/adobe/experience/marketo/marketo-contact-lead-person.schema.json

```diff
@@ -0,0 +1,190 @@
+  "meta:abstract": true,
+  "definitions": {
+    "marketo-contact-lead-person": {
+      "properties": {
+        "marketo:marketoIntegrationLogID": {
+          "title": "Marketo Integration Log Id",
+          "description": "Marketo integration log id.",
+          "type": "string"
+        },
+        "marketo:additionalFields": {
+          "title": "Additional Fields",
+          "description": "Additional fields.",
+          "type": "string"
+        },
+        "marketo:anonymousHostname": {
+          "title": "Anonymous Hostname",
+          "description": "Anonymous hostname.",
+          "type": "string"
+        },
+        "marketo:blackListed": {
+          "title": "Black Listed Indicator",
+          "description": "Black listed indicator.",
+          "type": "boolean"
+        },
+        "marketo:blackListedCause": {
+          "title": "Black Listed Cause",
+          "description": "Black listed cause.",
+          "type": "string"
+        },
+        "marketo:emailInvalid": {
+          "title": "Invalid Email Indicator",
+          "description": "Invalid email indicator.",
+          "type": "boolean"
+        },
+        "marketo:emailInvalidCause": {
+          "title": "Email Invalid Cause",
+          "description": "Email invalid cause.",
+          "type": "string"
+        },
+        "marketo:employeePositionID": {
+          "title": "Employee Position Id",
+          "description": "Employee position id.",
+          "type": "string"
+        },
+        "marketo:inferredAddress": {
+          "title": "Inferred Address",
+          "description": "Address details inferred by reverse IP lookup of the lead’s first recorded web visit.",
+          "$ref": "https://ns.adobe.com/xdm/common/address"
+        },
+        "marketo:inferredCompany": {
+          "title": "Inferred Company",
+          "description": "Company name inferred by reverse IP lookup of the lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:inferredCompanyStatus": {
+          "title": "Inferred Company Status",
+          "description": "Status of the company name inferred by reverse IP lookup of the lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:inferredMetropolitanArea": {
+          "title": "Inferred Metropolitan Area",
+          "description": "Metropolitan area inferred by reverse IP lookup of lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:inferredPhoneAreaCode": {
+          "title": "Inferred Phone Area Code",
+          "description": "Phone area code inferred by reverse IP lookup of lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:anonymous": {
+          "title": "Anonymous Indicator",
+          "description": "Anonymous status of lead record. System managed.",
+          "type": "boolean"
+        },
+        "marketo:customer": {

```

Comment:
isCustomer?

Reply:
+1

Reply:
Initially the Salesforce boolean attributes were defined with "Is" prefix but later removed as the boolean flags already existing on XDM do not have any prefix like "Is" just to be consistent.

Reply:
Isn't this already name spaced as the marketo customer?